src/nxdoc/nuxeo-server/administration/configuration-parameters-index-nuxeoconf.md

```diff
@@ -2363,6 +2363,21 @@ Property to override the default Content Security Policy.
 
 **Since LTS 2023.16**
 
+* * *
+
+#### `org.nuxeo.web.ui.pagination.nuxeoSelectOptions.listingMaxItems`
+
+Allow user to set the max number of label options in nuxeo select for pagination.
```

Comment:
Add more description where in if the number of options exceed the max limit, then the nuxeo-select will not be visible.

Reply:
ok